mu4/scores/qml/MuseScore/Scores/NewScoreDialog.qml

```diff
@@ -56,8 +56,6 @@ QmlDialog {
                 spacing: 20
 
                 FlatButton {
-
-                    width: 40
```

Comment:
Please revert all changes in NewScoreDialog.qml and ScoresModule.qml. We will get conflicts